spec/javascripts/ChangePassword/ChangePassword.spec.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import {ChangePassword} from 'ChangePassword/index.jsx'
+
+it('should render itself', () => {
+  const props = {
+    lostPasswordToken: 'foo',
+    url: 'foo/bar',
+    errors: []
+  }
+  const wrapper = mount(<ChangePassword {...props} />)
+  expect(wrapper).toMatchSnapshot()
+})
+
+it('should render with errors when present', () => {
+  const props = {
+    lostPasswordToken: 'foo',
+    url: 'foo/bar',
+    errors: [{type: 'error', message: 'Ooops!'}]
+  }
+  const wrapper = mount(<ChangePassword {...props} />)
+  expect(wrapper.find('.pf-c-form__helper-text').text()).toContain('Ooops!')
```

Comment:
```suggestion
  expect(wrapper).toMatchSnapshot()
```

Reply:
I don't want to compare the snapshots, but to be sure that errors are rendered

Reply:
Snapshots are generated for each set of properties. In this case the snapshot with errors will have a `pf-c-form__helper-text` and in the future is something changes, it will detect it. Same kind of situation [here](https://github.com/3scale/porta/blob/af1095b5a63b3f719f46239b4f4823dcc64c2e18/spec/javascripts/NewService/formElements/ServiceManualListItems.spec.jsx#L20).

Reply:
Snapshots are our fastest way to assert any structure given certain props. A cost effective shield that will guard us for any other change in the component